features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -923,10 +923,11 @@
 <!--        <bundle>mvn:org.openhab.addons.bundles/org.openhab.extensionservice.marketplace.automation/${project.version}</bundle>-->
     </feature>
 
-    <feature name="openhab-misc-mqttbroker" description="Embedded MQTT Broker" version="${project.version}">
+    <feature name="openhab-misc-mqttbroker" description="MQTT Broker Moquette" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-mqtt</feature>
-        <bundle>mvn:org.openhab.io/org.openhab.io.mqttembeddedbroker/${project.version}</bundle>
+        <bundle dependency="true" start-level="80">mvn:io.moquette/moquette-broker/0.12.1</bundle>
```

Comment:
What is the default `start-level`? Shouldn't this be available before the ` org.openhab.io.mqttembeddedbroker`?

Reply:
I thought `dependency="true"` is making this sure

Reply:
I‘m not an expert on this, but as far as I understood `dependency=true` only relates to the installation of that bundle (if not there, install it) but not to the start order. Could be wrong.

Reply:
I'm also not sure. "start-level" implies an osgi bundle, but moquette is not an osgi bundle, just a library that needs to be in the classpath for the mqttembeddedbroker bundle